adminapi/parse.py

```diff
@@ -10,6 +10,9 @@
 
 
 def parse_query(term, hostname=None):  # NOQA: C901
+    # Ignore newlines to allow queries across multiple lines
+    term = term.replace('\n', '')
```

Comment:
Normally, in terminals or the like, new lines are seen as spaces, no? 
So, would it make sense to use that here too? To not replace a new line with nothing but with a space?
```suggestion
    term = term.replace('\n', ' ')
```

Reply:
I know the behavior from HTML. I actually tested pasting a query separated by newline to the Servershell search input and it acted as you described.

While implementing it I thought about replacing it with a whitespace as you suggested but couldn't find any example where a terminal would behave like replacing a newline with a whitespace and therefore rather sticked with the above.

Thus my curiosity about an example.